.evergreen/csfle/azurekms/create-and-setup-vm.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o pipefail
+set -o nounset
+
+if [ -z "${AZUREKMS_VMNAME_PREFIX:-}" ] || \
+   [ -z "${AZUREKMS_CLIENTID:-}" ] || \
+   [ -z "${AZUREKMS_TENANTID:-}" ] || \
+   [ -z "${AZUREKMS_SECRET:-}" ] || \
+   [ -z "${AZUREKMS_DRIVERS_TOOLS:-}" ] || \
+   [ -z "${AZUREKMS_RESOURCEGROUP:-}" ] || \
+   [ -z "${AZUREKMS_PUBLICKEYPATH:-}" ] || \
+   [ -z "${AZUREKMS_PRIVATEKEYPATH:-}" ] || \
+   [ -z "${AZUREKMS_SCOPE:-}" ]; then
+    echo "Please set the following required environment variables"
+    echo " AZUREKMS_VMNAME_PREFIX to an identifier string no spaces (e.g. CDRIVER)"
+    echo " AZUREKMS_CLIENTID"
+    echo " AZUREKMS_TENANTID"
+    echo " AZUREKMS_SECRET"
+    echo " AZUREKMS_DRIVERS_TOOLS"
+    echo " AZUREKMS_PUBLICKEYPATH"
+    echo " AZUREKMS_PRIVATEKEYPATH"
+    echo " AZUREKMS_SCOPE"
+    exit 1
+fi
+
+# Set defaults.
+export AZUREKMS_IMAGE=${AZUREKMS_IMAGE:-"Debian"}
+# Install az.
+"$AZUREKMS_DRIVERS_TOOLS"/.evergreen/csfle/azurekms/install-az.sh
+# Login.
+"$AZUREKMS_DRIVERS_TOOLS"/.evergreen/csfle/azurekms/login.sh
+# Create VM.
+. "$AZUREKMS_DRIVERS_TOOLS"/.evergreen/csfle/azurekms/create-vm.sh
+export AZUREKMS_VMNAME="$AZUREKMS_VMNAME"
+echo "AZUREKMS_VMNAME: $AZUREKMS_VMNAME" > testazurekms-expansions.yml
+# Assign role.
+"$AZUREKMS_DRIVERS_TOOLS"/.evergreen/csfle/azurekms/assign-role.sh
+# Install dependencies.
+AZUREKMS_SRC="$AZUREKMS_DRIVERS_TOOLS/.evergreen/csfle/azurekms/remote-scripts/setup-azure-vm.sh" \
+AZUREKMS_DST="~/" \
```

Comment:
Quoting the `~/` will inhibit its expansion. Use `$HOME` instead.

Reply:
The `~/` is the path on the remote. It is not intended to be expanded here. Updated to `./` since it has the same effect.